app/src/main/java/com/hover/stax/utils/Utils.kt

```diff
@@ -264,4 +265,18 @@ object Utils {
         else
             UIHelper.flashMessage(c, c.getString(R.string.enable_call_permission))
     }
+
+    fun openSupportEmailClient(activity: Activity) {
```

Comment:
This is a near duplicate of openEmail above. Do we need both?

Reply:
Yeah. It's a duplicate. I've removed one.